models/Product.js

```diff
@@ -9,7 +9,7 @@ const ProductSchema = new mongoose.Schema(
     },
     price: {
       type: Number,
-      required: [true, 'Please provide product price'],
+      required: [true, 'Please provide product price'], // AKOSREVIEW: A little bit counterintuitive to have required field AND default
       default: 0,
```

Comment:
Generally, if a field is `required` then we don't need to also have a `default` (because we're already not letting the caller ever sae this schema with an empty value).

Reply:
@akosasante  thank you!!!